shared/common-adapters/plain-input.js.flow

```diff
@@ -0,0 +1,85 @@
+// @flow
+import * as React from 'react'
+import {type StylesCrossPlatform} from '../styles'
+import type {TextType} from './text'
+
+export type KeyboardType =
+  | 'default'
+  | 'email-address'
+  | 'numeric'
+  | 'phone-pad'
+  // iOS only
+  | 'ascii-capable'
+  | 'numbers-and-punctuation'
+  | 'url'
+  | 'number-pad'
+  | 'name-phone-pad'
+  | 'decimal-pad'
+  | 'twitter'
+  | 'web-search'
+  // Android Only
+  | 'visible-password'
+
+export type Props = {
+  autoFocus?: boolean,
+  className?: string,
+  disabled?: boolean,
+  // Resize in a flexbox-like fashion
+  flexable?: boolean,
+  maxLength?: number,
+  multiline?: boolean,
+  onBlur?: () => void,
+  onChangeText?: (text: string) => void,
+  onFocus?: () => void,
+  placeholder?: string,
+  rowsMin?: number,
+  rowsMax?: number,
+  style?: StylesCrossPlatform,
+  textType?: TextType,
+  type?: 'password' | 'text' | 'number',
+
+  /* Platform discrepancies */
+  // Maps to onSubmitEditing on native
+  onEnterKeyDown?: () => void,
```

Comment:
Might be worth naming this accurately and just calling it `onSubmitEditing` even on desktop